.github/workflows/aws-deployment.yml

```diff
@@ -0,0 +1,95 @@
+---
+    name: AWS Deployment
+
+    on:
+      workflow_dispatch:
+        inputs:
+            environment:
+                description: "Environment to deploy to"
+                default: "staging"
+                options:
+                    - staging
+                    - production
+                required: true
+                type: choice
+
+    env:
+      AWS_ACCOUNT_ID: ${{ vars.AWS_PUBLIC_DATA_RELEASES_ACCOUNT_ID }}
+      AWS_REGION: ${{ vars.AWS_DEFAULT_REGION }}
+      STAGING_CLOUDFRONT_DISTRIBUTION_ID: ${{ secrets.STAGING_CLOUDFRONT_DISTRIBUTION_ID }}
+      STAGING_S3_BUCKET: s3://staging.timelapse.allencell.org
+      PRODUCTION_CLOUDFRONT_DISTRIBUTION_ID: ${{ secrets.PRODUCTION_CLOUDFRONT_DISTRIBUTION_ID }}
+      PRODUCTION_S3_BUCKET: s3://timelapse.allencell.org
+
+    permissions:
+        id-token: write  # Required for requesting the JWT and OIDC
+        contents: write  # Required for actions/checkout and OIDC tokens
+
+    jobs:
+        build:
+          runs-on: ubuntu-latest
+
+          steps:
+            - name: Checkout
+              uses: actions/checkout@f43a0e5ff2bd294095638e18286ca9a3d1956744
+
+            - uses: actions/setup-node@60edb5dd545a775178f52524783378180af0d1f8
```

Comment:
@SeanLeRoy These build steps will presumably need adjusting for your workflow.

Reply:
Ah yeah this seems a bit different, I'll likely need to workshop this a bit. Does it just need to produce some build artifact at `./dist`?

Reply:
Yes - the deploy step picks them up from there (we could also change paths, if need be).

Reply:
How should I go about testing this - if there is nothing in the `./dist` will the build fail (as in, could I safely run this workflow to see if I am placing the files in the correct way and place)?

Reply:
https://github.com/AllenInstitute/aics-fms-file-explorer-app/pull/113